docs/topics/auth/customizing.txt

```diff
@@ -274,6 +274,15 @@ return ``False`` or an empty set (depending on the check performed).
 Conversely, it is expected that any backend that handles object permissions
 returns ``False`` if no object is passed.
 
+.. note::
+
+    Some people expect that object permissions are a superset of global
+    permissions. That is not the case by default. In order to achieve that, you
+    can add the
+    :class:`~django.contrib.auth.backends.DefaultObjectPermissionsBackend` to
+    :setting:`AUTHENTICATION_BACKENDS`. It will return all global permissions
+    when checking for object permissions, resulting in the expected behavior.
```

Comment:
So at this stage, I think this section just doesn't read. There's two topics: 

1. Handling object permissions with he shipped components. 
2. Creating a custom backend that handles object permissions.

Reply:
For this kind of thing I imagine the "Advance Beginner" — they obviously got past the basics since they're here. SO someone wanting to implement object permissions and needing to see an explanation of what's provided and what they then have to do.

This was my point about building the examples here (from the default to adding a custom backend).

> I think it is crucial to explain how this can actually be used.

Absolutely! That's what's kind of missing: "how do I use this new stuff?"

(If you want to mention `get_permission_object()`, perhaps after the custom backend, then that makes sense there.)